packages/core/src/hooks/useEditor.tsx

```diff
@@ -3,6 +3,8 @@ import {
   EditorCollector
 } from "../editor/useInternalEditor";
 import { ConnectorElementWrapper } from "@craftjs/utils";
+import { useMemo } from "react";
+import { NodeId } from "interfaces";
```

Comment:
Change this to `.../interfaces`, otherwise the type definition for `selectNode` will be lost.